draft-ietf-tls-subcerts.md

```diff
@@ -95,7 +97,7 @@ informative:
 --- abstract
 
 The organizational separation between operators of TLS and DTLS
-endpoints and the certification authority can create limitations.  For
+endpoints and the certification authority (CA) can create limitations.  For
```

Comment:
```suggestion
endpoints and the certification authority can create limitations.  For
```

This acronym isn't used elsewhere in the Abstract, and it is defined on first use in the body.